templates/contribute/style.md

```diff
@@ -565,3 +565,29 @@ Documentation strings for declarations are delimited with `/-- -/`.
 
 See our [documentation requirements](doc.html) for more suggestions
 and examples.
+
+### Deprecation
+
+Deleting, renaming, or changing declarations can cause downstreams projects
+that rely on these definitions to fail to compile. Any publicly exposed theorems and
+definitions that are being removed should be gracefully transitioned by keeping the
+old theorem statement with a `@[deprecated]` attribute, to warn downstream projects to
+transition away from deprecated definitions before they are deleted. Renamed theorems
+can use a deprecated `alias` to the new name, while in cases where the theorem is not
+being replaced, the deprecated theorem definition can be marked with an attribute, like so:
```

Comment:
I wonder what exactly "is not being replaced" means. What do you think about "does not have a direct replacement"? (Perhaps that's just my personal opinion, so I'd like to hear yours.)